src/engraving/libmscore/pitch.cpp

```diff
@@ -33,11 +36,13 @@ int PitchList::pitchOffset(int tick) const
     if (empty()) {
         return 0;
     }
-    PitchList::const_iterator i = upperBound(tick);
-    if (i == constBegin()) {
+
+    std::vector<int> ticks = mu::keys(*this);
+    auto it = std::upper_bound(ticks.cbegin(), ticks.cend(), tick);
+    if (it == ticks.cend()) {
         return 0;
     }
-    --i;
-    return i.value();
+    --it;
+    return at(*it);
```

Comment:
Replacing this with 
'''
    try {
        return at(*it);
    }
    catch (...) {
        return 0;
    }
'''
fixes the vtests crashes and all but one utest crash

Reply:
Easier/smaller possible fix:
```
    return mu::value(*this, (*it));
```